### PR TITLE
Edge case test

### PR DESCRIPTION
**Pull request by kaigomes7**

### CODE REVIEW

File: spec/feature/integration_spec.rb [hunk 1]
```diff
@@ -1312,6 +1326,274 @@
 	visit foreign_courses_tamu_courses_path
 	expect(page).to have_content('Software Engineering')
 	
-	
   end
 end
+
+RSpec.describe 'Creating request cases', type: :feature do
```
Comment (LanceAbuan) [inline spec/feature/integration_spec.rb:1332]: This looks good. Good job breaking up the tests, too.

**Review by LanceAbuan — APPROVED** [review]
Everything looks good to me.

File: spec/feature/integration_spec.rb (hunk 1 above)
Comment (kaigomes7) [reply spec/feature/integration_spec.rb:1332]: Thanks!
